### PR TITLE
User can login and logout

### PR DESCRIPTION
**Pull request by KyunginNa**
[User can log in and log out](https://www.pivotaltracker.com/story/show/176475944)
## User Story
```
As a User,
In order to have a secure user experience,
I would like to log out when I'm done with the page.
```
## Changes proposed in this pull request:
* Creating loginForm component
* Creating log out button in CategoryMenu component
* Adding login function and logout function in Authentication module

## What I have learned working on this feature:
* Mocking out delete request
* How to implement sign out function using jtockauth

## Screenshots (Client)
Screenshot 2021-01-17 at 11 24 49

### CODE REVIEW

File: src/modules/Authentication.js [hunk 1]
```diff
@@ -1,29 +1,63 @@
 import JtockAuth from 'j-tockauth'
+import store from '../state/store/configureStore'
```
Comment (salindholm) [inline src/modules/Authentication.js:2]: Good call to use store this way!

File: src/components/CategoryMenu.jsx [hunk 2]
```diff
@@ -1,109 +1,113 @@
-import React, { useState } from 'react'
-import { useDispatch, useSelector } from 'react-redux'
-import { Link } from 'react-router-dom'
-import { Menu, Segment } from 'semantic-ui-react'
-import { getArticles } from '../modules/getArticles'
-import { useTranslation } from 'react-i18next'
-import RegistrationForm from './RegistrationForm'
-import SubscriptionForm from './SubscriptionForm'
+import React, { useState } from "react"
```
Comment (salindholm) [inline src/components/CategoryMenu.jsx]: Please sort out your formatters so you all have either single or double quotes!

**Review by salindholm — COMMENTED** [review]
Good work, left some feedback!

File: src/modules/Authentication.js (hunk 1 above)
Comment (KyunginNa) [reply src/modules/Authentication.js:2]: Thank you! :)

File: src/components/CategoryMenu.jsx (hunk 2 above)
Comment (KyunginNa) [reply src/components/CategoryMenu.jsx]: Yes!

**Pull request merged**